doc/source/PreConfiguredSites.rst

```diff
@@ -137,15 +151,29 @@ NASA Discover SCU17
 
 The following is required for building new spack environments with any supported compiler on this platform.
 
-**NEEDS UPDATING**
-
 .. code-block:: console
 
    module purge
    module use /discover/swdev/gmao_SIteam/modulefiles-SLES15
    module use /discover/swdev/jcsda/spack-stack/scu17/modulefiles
-   module load ecflow/5.11.4
 
```

Comment:
same as above - only the parts that are needed to build new spack environments go here, whatever is needed to load existing spack environments (including the module purge /module use statements that maybe the same as the ones that remain in readthedocs) go into the wiki

Reply:
Ah will do!